src/app/screens/ConfirmKeysend/index.tsx

```diff
@@ -107,17 +106,16 @@ function Keysend(props: Props) {
   }
 
   return (
-    <div>
-      <PublisherCard
-        title={origin.name}
-        description={origin.description}
-        image={origin.icon}
-      />
-      <div className="py-4">
-        <Container maxWidth="sm">
-          {!successMessage ? (
-            <>
-              <div className="mb-8">
+    <div className="h-full flex flex-col overflow-y-auto no-scrollbar">
+      <div className="text-center text-xl font-semibold dark:text-white py-2 border-b border-gray-200 dark:border-neutral-500">
```

Comment:
As bumi mentioned below. Looks like the header-component HTML is the same for all screens. Just the text is changing, right? If that's the case we should move it into it's own component.